epoxy-compose/src/main/java/com/airbnb/epoxy/ComposeInterop.kt

```diff
@@ -11,27 +11,60 @@ import androidx.compose.ui.viewinterop.AndroidView
 class ComposeEpoxyModel(
     private val composeFunction: @Composable () -> Unit
 ) : EpoxyModelWithView<ComposeView>() {
+    private lateinit var keys: Array<Any>
+
     override fun buildView(parent: ViewGroup): ComposeView = ComposeView(parent.context)
 
     override fun bind(view: ComposeView) {
         super.bind(view)
         view.setContent(composeFunction)
     }
+
+    override fun equals(other: Any?): Boolean {
+        if (other === this) return true
+        if (other !is ComposeEpoxyModel) return false
+        if (this.hashCode() != other.hashCode()) return false
+
+        for (i in keys.indices) {
+            if (keys[i] != other.keys[i]) return false
```

Comment:
this will crash if the keys array is different sizes. use a built in array equality/hashcode function